src/components/List/List.tsx

```diff
@@ -41,13 +45,14 @@ const List: FC<ListProps> = forwardRef(
       ariaLabel,
       ariaDescribedBy,
       renderOnlyVisibleItems = false,
+      defaultFocusIndex = 0,
```

Comment:
I think it should be `focusIndex` and also be able to get updates (in the current implementation if the focused item gets changed from outside the inner state wouldn't know it)